adafruit_adxl34x.py

```diff
@@ -0,0 +1,213 @@
+
+Based on drivers by K. Townsend and Tony DiCola
+
+Implementation Notes
+--------------------
+
+**Hardware:**
+https://www.adafruit.com/product/1231
+
+**Software and Dependencies:**
+
+* Adafruit CircuitPython firmware for the supported boards:
+  https://github.com/adafruit/circuitpython/releases
+
+* Adafruit's Bus Device library: https://github.com/adafruit/Adafruit_CircuitPython_BusDevice
+"""
+
+from micropython import const
+from adafruit_bus_device import i2c_device
+try:
+    from struct import unpack
+except ImportError:
+    from ustruct import unpack
+__version__ = "0.0.0-auto.0"
+__repo__ = "https://github.com/adafruit/Adafruit_CircuitPython_ADXL34x.git"
+#pylint: disable=bad-whitespace
+_ADXL345_DEFAULT_ADDRESS         = const(0x53) # Assumes ALT address pin low
+
+# Conversion factors
+_ADXL345_MG2G_MULTIPLIER = 0.004 # 4mg per lsb
+_STANDARD_GRAVITY = 9.80665 # earth standard gravity
+
+_ADXL345_REG_DEVID               = const(0x00) # Device ID
+_ADXL345_REG_THRESH_TAP          = const(0x1D) # Tap threshold
+_ADXL345_REG_OFSX                = const(0x1E) # X-axis offset
+_ADXL345_REG_OFSY                = const(0x1F) # Y-axis offset
+_ADXL345_REG_OFSZ                = const(0x20) # Z-axis offset
+_ADXL345_REG_DUR                 = const(0x21) # Tap duration
+_ADXL345_REG_LATENT              = const(0x22) # Tap latency
+_ADXL345_REG_WINDOW              = const(0x23) # Tap window
+_ADXL345_REG_THRESH_ACT          = const(0x24) # Activity threshold
+_ADXL345_REG_THRESH_INACT        = const(0x25) # Inactivity threshold
+_ADXL345_REG_TIME_INACT          = const(0x26) # Inactivity time
+_ADXL345_REG_ACT_INACT_CTL       = const(0x27) # Axis enable control for [in]activity detection
+_ADXL345_REG_THRESH_FF           = const(0x28) # Free-fall threshold
+_ADXL345_REG_TIME_FF             = const(0x29) # Free-fall time
+_ADXL345_REG_TAP_AXES            = const(0x2A) # Axis control for single/double tap
+_ADXL345_REG_ACT_TAP_STATUS      = const(0x2B) # Source for single/double tap
+_ADXL345_REG_BW_RATE             = const(0x2C) # Data rate and power mode control
+_ADXL345_REG_POWER_CTL           = const(0x2D) # Power-saving features control
+_ADXL345_REG_INT_ENABLE          = const(0x2E) # Interrupt enable control
+_ADXL345_REG_INT_MAP             = const(0x2F) # Interrupt mapping control
+_ADXL345_REG_INT_SOURCE          = const(0x30) # Source of interrupts
+_ADXL345_REG_DATA_FORMAT         = const(0x31) # Data format control
+_ADXL345_REG_DATAX0              = const(0x32) # X-axis data 0
+_ADXL345_REG_DATAX1              = const(0x33) # X-axis data 1
+_ADXL345_REG_DATAY0              = const(0x34) # Y-axis data 0
+_ADXL345_REG_DATAY1              = const(0x35) # Y-axis data 1
+_ADXL345_REG_DATAZ0              = const(0x36) # Z-axis data 0
+_ADXL345_REG_DATAZ1              = const(0x37) # Z-axis data 1
+_ADXL345_REG_FIFO_CTL            = const(0x38) # FIFO control
+_ADXL345_REG_FIFO_STATUS         = const(0x39) # FIFO status
+
```

Comment:
Add a `# pylint: enable=bad-whitespace` here to turn that check back on since you disabled it "globally" up above.

Reply:
done